torch/csrc/distributed/rpc/script_call.h

```diff
@@ -31,6 +31,7 @@ class TORCH_API ScriptCall : public RpcCommandBase {
   bool hasOp() const;
   std::shared_ptr<Operator> op() const;
   bool hasQualifiedName() const;
+  // FIXME It's weird to return a const _value_, should this return a reference?
```

Comment:
Yep, returning const reference look better to me too.

Reply:
I'll change it then